src/index.ts

```diff
@@ -36,8 +41,11 @@ const getRev = (osVersion: string) => {
 	}
 };
 
-const isDevelopmentVersion = (version: string) => {
-	return /(\.|\+|-)dev/.test(version);
+const compareValues = <T>(valueA: T, valueB: T) => {
+	if (valueA === valueB) {
+		return 0;
+	}
+	return valueA > valueB ? 1 : -1;
```

Comment:
This probably works here, but technically it has a fun bug, because comparison in JavaScript does not follow your assumptions, annoyingly.

There's a few places you can hit this I think, but `NaN` is easiest:

```ts
NaN === NaN
// false
NaN > NaN
// false
NaN < NaN
// false
```

With this code `compareValues(NaN, NaN) === -1`. It should probably be `0` (try `>`, `<`, and call it 0 if they're both false).